employee.rb

```diff
@@ -0,0 +1,31 @@
+# require './department'
+
+class Employee
+
+attr_reader :name, :salary, :reviews, :satisfactory
+  def initialize(name, salary)
+    @name = name
+    @salary = salary
+    @reviews = []
+  end
+
+  def add_review(review)
```

Comment:
Intense!  An employee can have more than one review!  Okay, I'll roll with and see where you go.  :)